funnel/models/mixins.py

```diff
@@ -0,0 +1,153 @@
+# -*- coding: utf-8 -*-
```

Comment:
This file could be `video.py` since it only contains the video mixin. I'm assuming other mixins will be considered special functionality as well and will therefore not be in a generic `mixins.py`.